tests/Backend/Snowflake/CloneIntoWorkspaceTest.php

```diff
@@ -339,6 +340,39 @@ public function testTableAlreadyExistsShouldThrowUserError()
         } catch (ClientException $e) {
             $this->assertEquals('workspace.duplicateTable', $e->getStringCode());
         }
+
+        try {
+            // Invalid option combination
+            $workspaces->cloneIntoWorkspace($workspace['id'], [
+                'input' => [
+                    [
+                        'source' => $tableId,
+                        'destination' => 'Langs',
+                    ],
+                ],
+                'overwrite' => true,
+                'preserve' => false,
+            ]);
+            $this->fail('table should not be created');
+        } catch (ClientException $e) {
+            $this->assertEquals('workspace.loadRequestLogicalException', $e->getStringCode());
+        }
+
+        // third load of same table with preserve and force
+        $workspaces->cloneIntoWorkspace($workspace['id'], [
+            'input' => [
+                [
+                    'source' => $tableId,
+                    'destination' => 'Langs',
+                ],
+            ],
+            'overwrite' => true,
+            'preserve' => true,
+        ]);
+
+        $backend = new SnowflakeWorkspaceBackend($workspace);
+        $workspaceTableData = $backend->fetchAll('Langs');
+        $this->assertCount(5, $workspaceTableData);
```

Comment:
Někde nahoře chybí check na to, že to předtím nebylo taky 5 - tj. že se ten load reálně provedl.

Reply:
Ale jinak ty testy mi prošly.

Reply:
přihodil sem

Reply:
Tak tos otestoval, že se nic nestalo teda. Před importem tam bylo 5, teď je tam taky 5. Tak co se změnilo tím importem teda?

Reply:
Tady v tom testu, ale o ten počet vůbec nejde. To je jen check navíc. Jde o to že funguje `overwrite`. Počet sem hodil na konec jako sanity check nic víc, není to vůbec relevantní.

Reply:
No, ale to, že funguje ten test nijak neověří. Stačí aby cloneIntoWorkspace s overwrite potichu žralo exceptiony a nevíš že nefunguje.